### ISSUE TITLE
add support for skopeo for copying docker images between clusters without requiring docker socket access

### ISSUE THREAD

**Issue by jstrachan** [fabric8io/fabric8-pipeline-library#223]
background: http://rhelblog.redhat.com/2017/05/11/skopeo-copy-to-the-rescue/

related:

* https://trello.com/c/8nTMn9fV

**Comment by jstrachan** [comment]
wonder if we need to add it as a binary to the clients docker image?

**Comment by lpsantil** [comment]
@jstrachan Do you mean something like this?

https://github.com/redhat-cop/containers-quickstarts/tree/master/jenkins-slaves/jenkins-slave-image-mgmt

**Comment by jstrachan** [comment]
@lpsantil yes! I figure we should be able to add skopeo as a standard function to the library; then look at using it by default to move images between environments. 

Maybe the simplest solution is to add `skopeo` to our current [clients image](https://github.com/fabric8io/fabric8-pipeline-library/#clients-node) then its available OOTB in all pipeline build pods?
https://github.com/fabric8io/fabric8-pipeline-library/#understanding-how-it-works
